etuutt_bot/commands/role.py

```diff
@@ -63,33 +70,94 @@ async def get_less_than(self, interaction: Interaction[EtuUTTBot], n: int = 2):
 
     # Remove all users from a role
     @app_commands.checks.bot_has_permissions(manage_roles=True)
-    @app_commands.command(
-        name="removeall",
-        description="Prend toutes les personnes ayant le rôle et leur retire.",
-    )
+    @app_commands.command(name="removeall")
     async def remove_all(self, interaction: Interaction[EtuUTTBot], role: discord.Role):
+        """Prend toutes les personnes ayant le rôle et leur retire.
+
+        Args:
+            interaction:
+            role: le rôle auquel qu'on veut retirer à tout le monde
```

Comment:
```suggestion
            role: le rôle qu'on veut retirer à tout le monde
```
J'avais pas vu la typo